database/mocks/articles.js

```diff
@@ -13,6 +13,7 @@ module.exports = [
     body:
       'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla faucibus ipsum non metus finibus ultricies. Donec ac auctor dui, sed fringilla est. Duis et pellentesque nisl, a gravida felis. Ut tempor felis id dignissim congue. Nunc blandit nunc sit amet dui pharetra, quis porttitor sem ullamcorper. Suspendisse faucibus imperdiet lacinia.',
     status: 'published',
+    readingTime: '1 min',
```

Comment:
Are you adding readingTime manually, or this is response body?

Reply:
No, these are mocks for seeding our database.